x/bank/msgs.go

```diff
@@ -21,7 +24,7 @@ func NewMsgSend(in []Input, out []Output) MsgSend {
 
 // Implements Msg.
 // nolint
-func (msg MsgSend) Route() string { return "bank" } // TODO: "bank/send"
+func (msg MsgSend) Route() string { return MsgRoute }
 func (msg MsgSend) Type() string  { return "send" }
```

Comment:
Should we also make `"send"` a constant?

Reply:
Hmm I don't think that's necessary, it won't be used elsewhere and that isn't standard for other modules (e.g. `x/stake/types/msg.go`